packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -1651,9 +1659,6 @@ class AccountInternal extends PureComponent<
         balances={balances}
         showBalances={showBalances}
         filtered={transactionsFiltered}
-        collapseTransactions={ids =>
-          this.props.splitsExpandedDispatch({ type: 'close-splits', ids })
-        }
       >
```

Comment:
With the change from a callback to a `useEffect`, I now see a very brief flash of expanded preview transactions on desktop. It's not a big deal, but any ideas on how we can avoid that?

Reply:
Should be fixed now :)